### PR TITLE
TASK-41619 : Synchronization AD - User status

### PR DESCRIPTION
**Pull request by GouadriaHanen**
added enabled to attributes while retrieving users from AD

### CODE REVIEW

File: exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java [hunk 1]
```diff
@@ -831,7 +832,13 @@ private User importUser(String username, boolean deleted, boolean updateModified
...
       // This is a mandatory treatment, thus it should not be surrounded by
       // try/catch block
+      if (externalUser.isEnabled()) {
       organizationService.getUserHandler().createUser(externalUser, true);
+      } else {
+        ((UserImpl) externalUser).setEnabled(true);
+        organizationService.getUserHandler().createUser(externalUser, true);
+        organizationService.getUserHandler().setEnabled(externalUser.getUserName(),false, true);
+      }
       internalUser = externalUser;
```
Comment (ahamdi) [inline exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java]:
```suggestion
      organizationService.getUserHandler().createUser(externalUser, true);
      if (!externalUser.isEnabled()) {
        organizationService.getUserHandler().setEnabled(externalUser.getUserName(), false, true);
      }
      internalUser = externalUser;
```

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java (hunk 1 above)
Comment (GouadriaHanen) [reply exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java]: while  creating a disabled user it throws a disabled user exception 
so we need to enable it, create it then disabled it

Comment (ahamdi) [reply exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java]: Ok please add a comment in the code to explain why we need to create the user enabled and then set him as disabled.

Comment (GouadriaHanen) [reply exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/externalstore/IDMExternalStoreImportService.java]: done!

**Review by ahamdi — DISMISSED** [review]

**Review by ahamdi — APPROVED** [review]

**Pull request merged**